salt/_pillar/velum.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+'''
+A module that adds data to the Pillar structure retrieved by an http request made to Velum
+
+Configuring the VELUM ext_pillar
+====================================
+
+Set the following Salt config to setup Foreman as external pillar source:
```

Comment:
"Foreman"? Is that some leftover from a copy & paste?

Reply:
Ouch, copied the structure (laziness) and overlooked that! Will fix, thanks!